src/helper.jsx

```diff
@@ -0,0 +1,4 @@
+
+export const findTabById = (tabs, selectedTabId) => (
+  tabs.find(({ id }) => id === selectedTabId)
+);
```

Comment:
https://github.com/mate-academy/react_tabs-js/pull/266#discussion_r1315095487
```suggestion
export const findTabById = (tabs, selectedTabId) => (
  tabs.find(({ id }) => id === selectedTabId) || tabs[0]
);
```